Composer/cypress/integration/CreateNewBot.spec.ts

```diff
@@ -1,35 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-context('Creating a new bot', () => {
```

Comment:
can this be replaced with an integration test on the new creation experience?

Reply:
Yeah I am working on one now

Reply:
So I added a simple integration test for testing the creation of empty bot e2e. It does have to wait a bit since the creation takes a bit prior to asserting that the bot was created successfully. I suppose we could mock the server response and bot creation but that would probably defeat the purpose of the integration test a bit.

Reply:
new creation Integration test runs fine locally but breaking the build on the CI machine, I am unable to see the logs to see what the CI machine has issues with when running the test, the logs link to a 404 page. 

My guess is the CI machines do not have node? Or perhaps the amount of time the UT waits prior to asserting on the post creation view is not long enough since the CI machine is installing the template each time from npm which takes a bit the first time it is run. I dont think it is that though since I have played with large wait times and the CI machine still fails. 

I need to figure out how best to run this test as it is the only blocker from getting this merged